codegenerator/cli/src/cli_args/init_config.rs

```diff
@@ -124,26 +124,26 @@ pub mod evm {
                                 }
                             };
 
-                            Network {
+                            Chain {
                                 id: selected_chain.network.get_network_id(),
                                 hypersync_config: None,
                                 rpc_config: None,
                                 rpc,
                                 start_block: selected_chain.network.get_start_block(),
                                 end_block,
                                 confirmed_block_threshold: None,
-                                contracts: Vec::new(),
+                                contracts: Some(Vec::new()),
                             }
                         });
 
-                    let contract = NetworkContract {
+                    let contract = ChainContract {
                         name: selected_contract.name.clone(),
                         address,
                         config: config.clone(),
                         start_block: None,
                     };
 
-                    chain.contracts.push(contract);
+                    chain.contracts.get_or_insert_with(Vec::new).push(contract);
```

Comment:
try run cargo clippy, it will likely suggest you use `get_or_default().push(contract)` or something like that.